### ISSUE TITLE
Individual paramters for federated learning

### ISSUE THREAD

**Issue by darpitdavetamu** [tensorflow/federated#2582]
### My custom-loss function
 def custom_loss(keras.losses.loss):

      loss_1 = K.mean(K.square((self.y_pred - self.y_true)/self.y_true))

      loss_2 = K.mean(K.square(self.y_pred - self.y_true))

     final_loss = \alpha * loss_1 + (1 - alpha) * loss_2

      return final_loss

Can I pass different alpha (hyperparameter) for individual users in tensorflow_federated?
